fpga/source/generated/palette_ram/constraints/palette_ram.ldc

```diff
@@ -17,9 +17,10 @@ set OUTPUT_CLK_EN 0
 set RESETMODE "sync"
 set BYTE_ENABLE 1
 set BYTE_WIDTH 2
+set BYTE_SIZE 8
 set ECC_ENABLE 0
 set INIT_MODE "mem_file"
-set INIT_FILE "misc/palette_ram_palette_ram_copy.mem"
+set INIT_FILE "D:/_Dev/Radiant/vera/fpga/source/generated/palette_ram/misc/palette_ram_palette_ram_copy.mem"
```

Comment:
I'm way out of my depth here, but aren't the four absolute paths in this PR a problem for others to build this?